docs/running.rst

```diff
@@ -211,6 +211,28 @@ Default: ``socialhome.local``
 
 Must be set to your Socialhome instance domain. Used for example to generate outbound links.
 
+SOCIALHOME_ADDITIONAL_APPS
```

Comment:
I hate to be picky but it's really important to keep the settings in alphabetical order. We're going to have many of them, and it will be annoying to scroll if they're not in good order. Order is more important IMHO than grouping. So SOCIALHOME_DOMAIN after SOCIALHOME_ADDITIONAL_APPS_URLS and SOCIALHOME_HOME_VIEW after SOCIALHOME_HTTPS 👍

Reply:
Wait, ignore the latter, O is before T of course 😆 But the first part is relevant, A before D.